src/components/Tooltip/Tooltip.tsx

```diff
@@ -189,7 +205,9 @@ export const Tooltip: FC<TooltipProps> = React.memo(
       );
 
       const handleReferenceClick = (event: React.MouseEvent): void => {
-        event.stopPropagation();
+        if (type !== TooltipType.Default) {
+          event.stopPropagation();
```

Comment:
so, if You put the tooltip on a clickable element will it still trigger the click event on touch? I'm guessing yes based on your attached videos, but I'm not 100%

Reply:
Will double check a console log is triggered on the element click. The video only represents the visible state, not the event handling.

Reply:
Made some updates, we are good on devices. Tooltip and Popup reference element onClick behave as expected without this change. Reverting and also applying to `handleReferenceKeyDown`